instrumentation/README.md

```diff
@@ -35,3 +35,11 @@ Additionally the following guidelines for package composition need to be followe
    This documentation SHOULD be in a dedicated `doc.go` file if the package is more than one file.
    It SHOULD contain useful information like what the purpose of the instrumentation is, how to use it, and any compatibility restrictions that might exist. 
 - Examples of how to actually use the instrumentation SHOULD be included.
+
+## Additional Instrumentation Packages
+
+Below are additional instrumentation packages outside of the opentelemetry-go-contrib repo:
+
+|  Package Name  |                     Link                    |                                                          Notes                                                                 |
+| :------------: | :-----------------------------------------: | :----------------------------------------------------------------------------------------------------------------------------: |
+| go-redis/redis | [Github](https://github.com/go-redis/redis) | Trace only; add the hook found [here](https://github.com/go-redis/redis/blob/master/redisext/otel.go) to your go-redis client. |
```

Comment:
We should probably use the fully qualified package name for the instrumentation: `github.com/go-redis/redis/v8/redisext`

It might also be helpful to structure the table with the package name linking to the source and the next column to the docs over at https://pkg.go.dev/. Maybe something like:

```suggestion
|  Package Name  | Documentation | Notes |
| :------------: | :-----------: | :---: |
| [`github.com/go-redis/redis/v8/redisext`](https://github.com/go-redis/redis/blob/v8.0.0-beta.5/redisext/otel.go) | [Go Docs](https://pkg.go.dev/github.com/go-redis/redis/v8@v8.0.0-beta.5.0.20200614113957-5b4d00c217b0/redisext?tab=doc) | Trace only; add the hook found [here](https://github.com/go-redis/redis/blob/master/redisext/otel.go) to your go-redis client. |
```

Reply:
This makes perfect sense. Should we maybe use the master branch for the links so that it doesn't fall out of date as quickly?

Reply:
Yeah, it is a bit of a trade off. Using the master branch has the downside of upstream changes causing the link to potentially break if the upstream directory changes. Linking to a specific hash or tag also helps give a clue as to what we see currently when making the doc, kind of like a timestamp.

I'm fine with either though, up to you.

Reply:
This is a great point - I stuck with the tagged links to avoid breaking changes